src/components/FileTreeNode/index.tsx

```diff
@@ -204,10 +207,26 @@ export class FileTreeNodeBase<TreeNodeType> extends React.Component<Props> {
         title = getTitleForType(linterType, isFolder);
       }
 
-      nodeIcons = (
-        <span className={styles.nodeIcons}>
-          <FontAwesomeIcon icon={getIconForType(linterType)} title={title} />
-        </span>
+      nodeIcons.push(
+        <FontAwesomeIcon
+          fixedWidth
+          key="linter-icon"
+          icon={getIconForType(linterType)}
+          title={title}
+        />,
+      );
+    }
+
+    if (hasComments) {
+      nodeIcons.push(
+        <FontAwesomeIcon
+          fixedWidth
+          key="comments-icon"
+          flip="horizontal"
+          icon={['fas', 'comment-alt']}
+          size="1x"
+          title={gettext('This file has comments')}
```

Comment:
This ends up putting this title on folders as well as files. For linter notices we have a `getTitleForType` function which ensures that files are titled as files and folders are titled as folders, so maybe we need something similar here? Or maybe just a ternary right here would do the trick?